tools/train_classifier.py

```diff
@@ -124,6 +124,7 @@ def train_model(path_in, path_out, model_name, model_version, num_layers_to_fine
 
     # Find label names
     label_names = os.listdir(directories.get_videos_dir(path_in, 'train'))
+    label_names.sort()
```

Comment:
if possible  use `natural sorting` instead of basic sorting.  For that  we have a  library called `natsort`. You can find that in `testing.py` on line 49.